system/ui/lib/list_view.py

```diff
@@ -11,383 +9,289 @@
 from openpilot.system.ui.lib.toggle import Toggle, WIDTH as TOGGLE_WIDTH, HEIGHT as TOGGLE_HEIGHT
 from openpilot.system.ui.lib.widget import Widget
 
-ITEM_BASE_HEIGHT = 170
 LINE_PADDING = 40
-LINE_COLOR = rl.GRAY
+ITEM_BASE_HEIGHT = 170
 ITEM_PADDING = 20
-ITEM_SPACING = 80
 ITEM_TEXT_FONT_SIZE = 50
 ITEM_TEXT_COLOR = rl.WHITE
 ITEM_DESC_TEXT_COLOR = rl.Color(128, 128, 128, 255)
 ITEM_DESC_FONT_SIZE = 40
 ITEM_DESC_V_OFFSET = 140
-RIGHT_ITEM_PADDING = 20
 ICON_SIZE = 80
 BUTTON_WIDTH = 250
 BUTTON_HEIGHT = 100
-BUTTON_BORDER_RADIUS = 50
 BUTTON_FONT_SIZE = 35
-BUTTON_FONT_WEIGHT = FontWeight.MEDIUM
 
-TEXT_PADDING = 20
+
+# Type Aliases for Clarity
+_StrSrc = str | Callable[[], str] | None
+_BoolSrc = bool | Callable[[], bool]
```

Comment:
global variables feel cleaner without the underscore, even if they're local for now